docs/advanced/index.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: index
+title: Advanced
+navigation_weight: 1
+---
+
+# Caching CDN Requests or Non-200 Responses
+
+If an attempt is made to cache an asset that is on a different origin or
+the asset returns a non-200 response, these modules will throw an error. The
+reason for this default behavior is to ensure that only good and local
+responses are cached.
+
+Depending on how you are using these libraries will depend on how you can add
+support for external resources.
+
```

Comment:
Having depending/depend is consfusing. Try something like: 'Use of these libraries depends on how you support external resources.'.